travis_configure.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+# Script for setting up OS specific CI test environment on travis-ci.org
+#
+# Copyright (C) 2017 Andreas Bok Andersen <andreas.bok@sociomantic.com>
+#
+
+set -e
+
+# Script which is run before installation script is called
+#
+# Installs fpm needed for deb packaging
+#
+# Returns 0
+before_install()
+{
+    if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+        if [[ "$DIST_PACKAGE_TARGET" == "DEB" ]]; then
+            gem install fpm
+        fi
+    fi
+
+    return 0
+}
+
+# Enable binary logging by setting log_bin and server_id in the [mysqld]
+# section of the mysql config file
+#
+# $1 path to mysql config file
+#
+# Returns 0
+enable_mysqlbinlog()
+{
+    sudo sed -i -e 's/^# *server.id.*$/server_id=1/g' $1
+    sudo sed -i -e 's/^# *log.bin.*$/log_bin=mysql-bin/g' $1
+
+    return 0
+}
+
+# Invoked by the travis hook before_script
+#
+# - prepares the build environment
+#
+# Returns 0
+before_script()
+{
+    if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+        if [[ "$DIST_PACKAGE_TARGET" == "DEB" ]]; then
+            # Ensures MySQL is started with binlog enabled. This is done by
+            # modifying the default my.cnf file.
+            #  1. Enable replication in mysql config file
+            #  2. Restart MySQL server
+            enable_mysqlbinlog "/etc/mysql/my.cnf"
+            sudo service mysql restart
+            mysql -u root -e "SET PASSWORD FOR 'root'@'localhost' = PASSWORD('')"
+        elif [[ "$DIST_PACKAGE_TARGET" == "RPM" ]]; then
+            # build a docker image with Centos 7 and MySQL
+            docker build -t centos7_mysql56 -f docker/Dockerfile .
```

Comment:
In this case make it explicit with a comment. The underlying concern was that, if you want to add an architecture / test, you don't want to miss out on the configuration somewhere (although it will most likely be visible if you do :) )